Branch-SDK/Branch-SDK/ContentDiscoveryManifest.m

```diff
@@ -0,0 +1,107 @@
+//
+//  ContentDiscoverManifest.m
+//  Branch-TestBed
+//
+//  Created by Sojan P.R. on 8/18/16.
+//  Copyright © 2016 Branch Metrics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ContentDiscoveryManifest.h"
+#import "BNCPreferenceHelper.h"
+#import "ContentPathProperties.h"
+#import <UIKit/UIKit.h>
+#import "BranchConstants.h"
+
+@implementation ContentDiscoveryManifest
+
+NSString *manifestVersion;
```

Comment:
also please declare this variable up in the private interface as I described above.